clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -301,6 +301,24 @@ public void testInvalidKeystorePassword() throws Exception {
             // Expected exception
         }
     }
+
+    /**
+     * Tests that client connections can be created to a server
+     * if null truststore password is used
+     */
+    @Test
+    public void testNullTruststorePassword() throws Exception {
```

Comment:
@ijuma The password is used to check the integrity of the truststore. If password is not specified, the integrity check is not performed, but the truststore can still be read and used.